src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallLibertyTask.groovy

```diff
@@ -15,10 +15,14 @@
  */
 package net.wasdev.wlp.gradle.plugins.tasks
 
+import javax.xml.parsers.*
+
 import org.gradle.api.Project
 import org.gradle.api.tasks.TaskAction
 
 class InstallLibertyTask extends AbstractTask {
+    
+    final MAVEN_REPO = 'http://repo1.maven.org/maven2/' 
```

Comment:
I added the code to download from the maven repositories(internal or remote), in case the file is not downloaded by gradle dependencies programmatically. For instance,  if user doesn't specify maven repository in the gradle script, it won't know about the maven repo. 

- if maven repository is added in the script, it will download Liberty first into the gradle repo then it will use the file from the gradle repo. 
- if maven repository is not added in the script, it will check gradle repository first then maven repositories.